docs/package_docs.rst

```diff
@@ -86,52 +86,101 @@ We suggest this template:
 
    See LICENSE and COPYRIGHT files.
 
-.. _pkg-doc-license:
-
-LICENSE
--------
-
-A ``LICENSE`` file in the repository's root is the canonical description of LSST's code licensing.
-
-The canonical source of this file's content is https://www.lsstcorp.org/LegalNotices/LsstLicenseStatement.txt, though ensure that it is named ``LICENSE`` in the repository.
-
 .. _pkg-doc-copyright:
 
 COPYRIGHT
 ---------
 
-We assert copyright information in a centralized ``COPYRIGHT`` file, located in the repository's root.
+We assert copyright information in a centralized ``COPYRIGHT`` file, located in the repository's root rather than attempting to track copyright in each file separately within a package.
 Using a ``COPYRIGHT`` file allows us to maintain copyright information more effectively than in source code comments.
 (See `RFC-45 <https://jira.lsstcorp.org/browse/RFC-45>`_ for background on this policy).
 
-Default Copyright
-^^^^^^^^^^^^^^^^^
+Some code written before construction officially started will have an LSST Corporation copyright and this should not be changed unless it has erroneously been used for code after construction began in August 2015.
```

Comment:
I would put this sentence about "old code" at the very end of this section. Otherwise, it's one of the first things a person sees.